Automation Examples/020.md

```diff
@@ -0,0 +1,161 @@
+# Process
+
+Open https://www.vat19.com/, go to fun & strange stuff and download first 20 products' information (name, price, image) and save it into excel file. Template is available [here](Products%20Template.xlsx).
+
+# Steps to automate
+
+#### 1. Specify a path to your folder, open website and maximise the browser
+
+```G1ANT
+♥basePath = C:\Users\dell\Desktop\Automation Examples
+selenium.open chrome url https://www.vat19.com/
```

Comment:
usage of selenium addon: not mentioned to include addon to robot.

Reply:
Grrrrr 😛